ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
+
+__all__ = ["SensitivityPointSource"]
+
+
+def convert_astropy_array(arr, unit=None):
+    """
+    converts a python list of quantities into a quantified numpy array of the same
+    dimension
+
+    Parameters
+    ----------
+    arr : python list
+        list of quantities of same dimension (not strictly exact same unit)
+    unit : astropy unit, optional (default: None)
+        if set, uses this as the unit of the numpy array; if not, uses the unit of the
+        first element
+        ought to be of same dimension of the quantities in the list (there is no test)
+
+    Returns
+    -------
+    a : quantified numpy array
+    """
+
+    if unit is None:
+        unit = arr[0].unit
+    return np.array([a.to(unit).value for a in arr])*unit
+
+
+def crab_source_rate(energy):
```

Comment:
Hmm, we have this already in Gammapy:
http://docs.gammapy.org/en/latest/api/gammapy.spectrum.CrabSpectrum.html

@kosack - Did we discuss if `import gammapy` from `ctapipe` is OK yet?
(Clearly my preference would be that we work together and don't duplicate stuff.)
If not possible / desired we can find a sane way to share code without duplicating much effort.